docs/builders/iso/xenserver-iso.html.markdown

```diff
@@ -154,6 +154,11 @@ each category, the available options are alphabetized and described.
   will shut down the VM gracefully through the Xen api's vm shutdown command. Unless
   you have special requirements this should typically be left to its default.
 
+* `shutdown_timeout` (string) - The amount of time to wait after executing
+  the `shutdown_command` for the virtual machine to actually shut down.
+  If it doesn't shut down in this time, it is an error. By default, the timeout
+  is "5m", or five minutes.
+
```

Comment:
Is shutdown_timeout still a thing?
I don't see it in master anymore, but it might still be usefull if one needs a shutdown command

Reply:
Upstream (github.com/xenserver/packer-builder-xenserver) had that config documented but I haven't seen any references aside from the documentation (https://github.com/xenserver/packer-builder-xenserver/search?q=shutdown_timeout). So I likely removed it because I believe the value is hardcoded [here](https://github.com/ddelnano/packer-plugin-xenserver/blob/a472fd2b02699a72f080ca08aaa483d6f27101e1/builder/xenserver/common/step_shutdown.go#L48).